invisible_cities/cities/beersheba.py

```diff
@@ -83,13 +83,16 @@
 from .. io. event_filter_io    import event_filter_writer
 from .. io.         kdst_io    import kdst_from_df_writer
 
+from .. types.ic_types         import NoneType
 from .. types.symbols          import HitEnergy
 from .. types.symbols          import InterpolationMethod
 from .. types.symbols          import CutType
 from .. types.symbols          import DeconvolutionMode
 
 from .. core                   import system_of_units as units
 
+from typing import Union
```

Comment:
Can this line be moved up to be with the other `typing` imports?

Reply:
Oops, yes, I didn't see them. They are usually at the end of the import block, I will move the other ones down as it is our convention.